src/ontology/lexmatch/README.md

```diff
@@ -3,49 +3,48 @@
 * mondo-only: Positive mappings in MONDO not caught by the lexical mapping pipeline
 * split-mapping-set: Unmapped mappings broken down by predicate_id
 ## Summary of mappings:
-## unmapped_xxxx_lex & unmapped_xxxx_lex_exact
- * Number of mappings in [`unmapped_icd_lex`](unmapped_icd_lex.tsv): 1928
```

Comment:
Can you sort the output on this file here so it does not get such a big diff each time? Its super useful when reviewing mondo-ingest runs to see the changed numbers!